_source/providenceConfiguration/mainConfiguration/media_processing_media_type.csv

```diff
@@ -0,0 +1,32 @@
+Key,Description,Example
+QUEUE,Queue to deliver the media type to,"``QUEUE = mediaproc``"
+QUEUED_MESSAGE,Message to show on queue listing,"``QUEUED_MESSAGE = _('Image is being processed')``"
+QUEUE_USING_VERSION,Version to use when queueing,"``QUEUE_USING_VERSION = original``"
```

Comment:
Should note here that you have to specify a version here that is not set to queue, and that you'd almost always want to be using "original" (aka. the uploaded file).